spec/index.html

```diff
@@ -14,6 +14,7 @@ <h1>Atomics.pause ( [ _iterationNumber_ ] )</h1>
     1. If _iterationNumber_ is not *undefined*, then
       1. If _iterationNumber_ is not an integral Number, throw a *TypeError* exception.
       1. If ℝ(_iterationNumber_) &lt; 0, throw a *RangeError* exception.
+    1. If the execution environment of the ECMAScript implementation supports a signal that the current executing code is in a spin-wait loop, send that signal. An ECMAScript implementation may send that signal multiple times, determined by _iterationNumber_ when not *undefined*. The number of times the signal is sent for an integral Number _N_ is at most the number of times it is sent for _N_ + 1.
```

Comment:
```suggestion
    1. [declared="N"] If the execution environment of the ECMAScript implementation supports a signal that the current executing code is in a spin-wait loop, send that signal. An ECMAScript implementation may send that signal multiple times, determined by _iterationNumber_ when not *undefined*. The number of times the signal is sent for an integral Number _N_ is at most the number of times it is sent for _N_ + 1.
```

alternatively, you can just name `iterationNumber`:

```suggestion
    1. If the execution environment of the ECMAScript implementation supports a signal that the current executing code is in a spin-wait loop, send that signal. An ECMAScript implementation may send that signal multiple times, determined by _iterationNumber_ when not *undefined*. The number of times the signal is sent is at most the number of times it is sent for any integral Number greater than _iterationNumber_.
```